SmartContract/package.json

```diff
@@ -2,6 +2,7 @@
   "dependencies": {
     "chai": "^4.3.4",
     "chai-as-promised": "^7.1.1",
-    "truffle-assertions": "^0.9.2"
+    "truffle-assertions": "^0.9.2",
```

Comment:
why did this get upadted?

Reply:
*updated

Reply:
The comma, because there's a new dependency in the next line. This file is auto generated, by the way